cmd/crossplane/core/core.go

```diff
@@ -113,6 +113,7 @@ type startCommand struct {
 	EnableUsages               bool `group:"Alpha Features:" help:"Enable support for deletion ordering and resource protection with Usages."`
 	EnableRealtimeCompositions bool `group:"Alpha Features:" help:"Enable support for realtime compositions, i.e. watching composed resources and reconciling compositions immediately when any of the composed resources is updated."`
 	EnableSSAClaims            bool `group:"Alpha Features:" help:"Enable support for using Kubernetes server-side apply to sync claims with composite resources (XRs)."`
+	EnableDependencyUpgrades   bool `group:"Alpha Features:" help:"Enable support for upgrading dependency versions when a package is upgraded."`
```

Comment:
I know this was what suggested [here](https://github.com/crossplane/crossplane/pull/5991#discussion_r1803069281), but `upgrade` vs `update` throughout the codebase is bugging me. Either have this as `EnableDependencyUpdates` or replace all occurances of `updates` in the PR with the `upgrades`